config-user-example.yml

```diff
@@ -37,10 +37,6 @@ output_dir: ./esmvaltool_output
 # Auxiliary data directory (used for some additional datasets)
 auxiliary_data_dir: ./auxiliary_data
 
-# Diagnostics create plots? [true]/false
```

Comment:
I think we may want to keep these here until the feature is actually removed in v2.4

Reply:
Having second thoughts about it: The current `config-user.yml` file gives this warning when used:

```
Warning: Using 'write_plots' in config-user.yml is deprecated and will be removed in ESMValCore version 2.4. For diagnostics that support this setting, it should be set in the diagnostic script section of the recipe instead. Remove the setting from config-user.yml to get rid of this warning message.
Warning: Using 'write_netcdf' in config-user.yml is deprecated and will be removed in ESMValCore version 2.4. For diagnostics that support this setting, it should be set in the diagnostic script section of the recipe instead. Remove the setting from config-user.yml to get rid of this warning message.
```

Do we want this?

Reply:
Yes, I think so, they clearly describe how to get rid of the warning. If we removed the setting from the example file now, several recipes may crash because they still need the setting.

Reply:
I don't think they would crash (since these options are still set), but fine for me!

https://github.com/ESMValGroup/ESMValCore/blob/a5209d88f02f2076d92505eaa4dbb3326af0948c/esmvalcore/_config/_config.py#L72-L73